plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -230,17 +238,14 @@ const entries = {
 		'wc-blocks': './assets/js/index.js',
 
 		// Blocks
-		...getBlockEntries( 'index.{t,j}s{,x}' ),
+		...getBlockEntries( 'index.{t,j}s{,x}', {
+			...blocks,
+			...cartAndCheckoutBlocks,
```

Comment:
We still need to generate cart and checkout entry points for the editor side bundle and the styling bundle as normal so we add them back into the entry list here.